packages/redux/src/users/closets/actions/factories/fetchUserClosetFactory.ts

```diff
@@ -0,0 +1,77 @@
+import * as actionTypes from '../../../actionTypes.js';
+import {
+  type Closet,
+  type ClosetSummary,
+  type Config,
+  type GetUserCloset,
+  type GetUserClosetQuery,
+  toBlackoutError,
+  type User,
+} from '@farfetch/blackout-client';
+import adaptProductImages from '../../../../helpers/adapters/adaptProductImages.js';
+import type { Dispatch } from 'redux';
+import type { FetchUserClosetAction } from '../../types/actions.types.js';
+import type {
+  GetOptionsArgument,
+  StoreState,
+} from '../../../../types/index.js';
+
+/**
+ * Method responsible for fetching an user specific closet.
```

Comment:
Same issue in https://github.com/Farfetch/blackout/pull/854/files#r1317006984

Reply:
and done!